precompiles/relay-encoder/src/lib.rs

```diff
@@ -79,30 +78,25 @@ where
 	Runtime: pallet_evm::Config,
 	Runtime::RuntimeCall: Dispatchable<PostInfo = PostDispatchInfo> + GetDispatchInfo,
 {
-	#[precompile::public("encodeBond(uint256,uint256,bytes)")]
-	#[precompile::public("encode_bond(uint256,uint256,bytes)")]
+	#[precompile::public("encodeBond(uint256,bytes)")]
+	#[precompile::public("encode_bond(uint256,bytes)")]
```

Comment:
What actually changed here? I don't see anything different, but GH is suggesting `uint256,` changed...